aries-site/src/layouts/content/AccessibilityTable.js

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable react/prop-types */
+import { Box, NameValueList, NameValuePair, Text } from 'grommet';
+import {
+  StatusCriticalSmall,
+  StatusGoodSmall,
+  StatusWarningSmall,
+  CircleInformation,
+} from 'grommet-icons';
+
+const StatusLabel = ({ icon: Icon, color, label }) => (
+  <Box align="center" direction="row" gap="xsmall">
+    <Icon color={color} />
+    <Text> {label}</Text>
+  </Box>
+);
+
+export const AccessibilityTable = ({ statuses = [] }) => {
+  // Calculate counts of each status
+  // revisit to clean up the code here
+  const calculateAccessibilityTestCounts = () => {
+    let conditional = 0;
+    let passed = 0;
+    let passedWithExceptions = 0;
+    let failed = 0;
+
+    statuses.forEach(status => {
+      if (status === 'passed') {
+        passed += 1;
+      } else if (status === 'passed with exceptions') {
+        passedWithExceptions += 1;
+      } else if (status === 'failed') {
+        failed += 1;
+      } else if (status === 'conditional') {
+        conditional += 1;
+      }
+    });
+
+    return { conditional, passed, passedWithExceptions, failed };
+  };
+
+  const { conditional, passed, passedWithExceptions, failed } =
+    calculateAccessibilityTestCounts();
+
+  return (
+    // room to clean up the code here can come back to this
+    // once we have the final design
+    <Box pad={{ vertical: 'medium' }} gap="medium">
+      <Box
+        pad={{ vertical: 'small', horizontal: 'medium' }}
+        alignSelf="start"
+        background="background-front"
+        round="small"
+      >
+        <NameValueList
```

Comment:
I like the NameValueList view, we can see what other people think though

Reply:
I agree with this as well but will also be curious to others point of view.